### ISSUE TITLE
bug: routerlink cause page reload

### ISSUE THREAD

**Issue by danielehrhardt** [ionic-team/ionic-framework#25078]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x

### Current Behavior

When you attach a routerLink to for example ion-card the Browser refresh on click. 

### Expected Behavior

The Page should not refresh on click. 

### Steps to Reproduce

Install the latest Version: 6.0.15 and add a routerLink to a Ionic Element. 

### Ionic Info

@ionic/core --> 6.0.15

### Additional Information

The Bug is introduced because of this line got removed:
https://github.com/ionic-team/ionic-framework/pull/25014/files#diff-ddccb39fd69c03734ed6d392993f4cd4308e32589fe53a2e5d29e0e7bb552743L47

**Comment by demsey2** [comment]
when I try routerlink with ion-item it reloads the whole page but when I change ion-item to div or a different element it works fine

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/25076#issuecomment-1092076779.

Please see a link for a fix timeline as well as a dev build that you can use to get unblocked.

**Comment by liamdebeasi** [comment]
This issue is resolved in Ionic 6.0.16. Please update your apps to receive a fix:

```
npm install @ionic/angular@6.0.16
```
